File: app/src/main/java/com/krystianwsul/checkme/notifications/TimeChangeReceiver.java
package com.krystianwsul.checkme.notifications;

import android.annotation.SuppressLint;
import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class TimeChangeReceiver extends BroadcastReceiver {
    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    @Override
    public void onReceive(Context context, Intent intent) {
        TickService.startService(context);
    }
}

File: app/src/main/java/com/krystianwsul/checkme/notifications/TimeChangeReceiver.kt
package com.krystianwsul.checkme.notifications

import android.annotation.SuppressLint
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class TimeChangeReceiver : BroadcastReceiver() {

    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    override fun onReceive(context: Context, intent: Intent) {
        TickJobIntentService.startServiceNormal(context, "TimeChangeReceiver")
    }
}
